test/alternator/test_ttl.py

```diff
@@ -655,9 +655,14 @@ def test_ttl_expiration_lsi_key(dynamodb, waits_for_expiration):
 # becoming expired. This event should contain be a REMOVE event, contain
 # the appropriate information about the expired item (its key and/or its
 # content), and a special userIdentity flag saying that this is not a regular
-# REMOVE but an expiration.
-@pytest.mark.veryslow
-def test_ttl_expiration_streams(dynamodb, dynamodbstreams):
+# REMOVE but an expiration. Reproduces issue #11523.
+def test_ttl_expiration_streams(dynamodb, dynamodbstreams, waits_for_expiration):
+    # Alternator Streams currently doesn't work with tablets, so until
+    # #23838 is solved, skip this test on tablets.
+    print(TAGS)
+    if '0' in str(TAGS):
+        pytest.skip("Streams test skipped on tablets due to #23838")
```

Comment:
Does this skip work correctly? Currently, the `TAGS` global variable is set via this fixture:

```python
@pytest.fixture(params=[
    [{'Key': 'experimental:initial_tablets', 'Value': 'none'}],
    [{'Key': 'experimental:initial_tablets', 'Value': '0'}],
], ids=["using vnodes", "using tablets"], autouse=True)
def tags_param(request):
    # Set TAGS in the global namespace of this module
    global TAGS
    TAGS = request.param
```

So it looks like, to me, that we should iterate over the elements of the TAGS list, find the dict `d` for which `d['Key'] == 'experimental:initial_tablets'`, and then skip the test if `d['Value'] == '0'`.

We don't run Alternator tests for tablets yet, do we?

Reply:
Yes, I'm thinking of all that TAGS business - to run both vnodes and tablets - as something temporary - and even more so this "skipping" of the tablets case - so I didn't put too much thinking into it. 

The "0" is meant to look for the `{'Key': 'experimental:initial_tablets', 'Value': '0'}` option in TAGS - which means tablets, while not matching `{'Key': 'experimental:initial_tablets', 'Value': 'none'}` which means vnodes.

Anyway, I'll spend a few more minutes on this and change it to check for a numeric value.